shared/middleware/helpers/auth_helpers.rb

```diff
@@ -29,6 +30,13 @@ def under_13?
   age < 13
 end
 
+# Returns the sharing_disabled property of a user with a given user_id
+def get_user_sharing_disabled(user_id)
+  user_properties = DASHBOARD_DB[:users].where(id: user_id).pluck(:properties)
+  parsed_properties = JSON.parse(user_properties[0])
+  !!parsed_properties["sharing_disabled"]
+end
```

Comment:
this method should `return true unless user_properties` to guard against empty properties, since the `properties` column appears to be optional in the `users` table.